vshn/website
# feat: add Reusable components

**Describe what changes this pull request brings**
- This pull request brings the Reusable components.


**Describe any additional changes**
N/A


**Steps to test**
1. Open [the page](https://reusable-components--vshn.netlify.app/development-page/)
2. Confirm that everything looks and works as expected


**Screenshots**
![page_full_01-Apr-2021_12-57-39](https://user-images.githubusercontent.com/48465000/113277890-e3187c00-92e9-11eb-88cb-795f0a9c44f0.png)



**Checklist**



**TODO**
- [x] Integrate components with WP 
- [x] Implement the responsive styles  


**References**
 - [Figma](https://www.figma.com/file/fd09Cpdrkt3bVSgyVkzqym/VSHN-Website-Design?node-id=3833%3A22299)
 - [Proposal to Create Lazy Blocks for some Pages](https://www.notion.so/Proposal-to-Create-Lazy-Blocks-for-some-Pages-613a8a6e1975431d9d34f9979e01706b)
